src/main/java/com/softserve/academy/spaced/repetition/domain/Deck.java

```diff
@@ -31,12 +31,21 @@
     @JoinColumn(name = "user_id")
     private User deckOwner;
 
-    @ManyToMany(cascade = CascadeType.ALL)
+    @OneToMany(cascade = CascadeType.ALL)
     @JoinTable(name = "deck_cards", joinColumns = {
```

Comment:
I don't think that the @JoinTable annotation is needed when an @OneToMany relationtship is used.

Reply:
You propose to save data about cards of some deck in table of decks?

Reply:
No, of course. no. But you don't need an additional table like "deck_cards" when using OneToMany relationship. In this case you have to use a foreign key in the table named Card.

Reply:
I'll think about it and try to make changes.